specification/cognitiveservices/data-plane/Face/v1.0/Face.json

```diff
@@ -0,0 +1,2655 @@
+      "schema": {
+       "$ref": "#/definitions/APIError"
+      }
+     }
+    },
+    "produces": [
+     "application/json"
+    ]
+   },
+   "patch": {
+    "description": "Update an existing person group's display name and userData. The properties which does not appear in request body will not be updated.",
+    "operationId": "PersonGroups_Update",
+    "parameters": [
+     {
+      "name": "personGroupId",
+      "in": "path",
+      "description": "personGroupId of the person group to be updated.",
+      "required": true,
+      "type": "string"
+     },
+     {
+      "$ref": "#/parameters/SubscriptionKey"
+     },
+     {
+      "name": "body",
+      "in": "body",
+      "required": true,
+      "x-ms-client-flatten": true,
+      "schema": {
+       "$ref": "#/definitions/CreatePersonGroupRequest"
+      }
+     }
+    ],
+    "consumes": [
+     "application/json"
+    ],
+    "responses": {
+     "200": {
+      "description": "A successful call returns an empty response body."
+     },
+     "default": {
+      "description": "Error response.",
+      "schema": {
+       "$ref": "#/definitions/APIError"
+      }
+     }
+    },
+    "produces": [
+     "application/json"
+    ]
+   }
+  },
+  "/persongroups/{personGroupId}/training": {
+   "get": {
+    "description": "Retrieve the training status of a person group (completed or ongoing).",
+    "operationId": "PersonGroups_GetTrainingStatus",
+    "parameters": [
+     {
+      "name": "personGroupId",
+      "in": "path",
+      "description": "personGroupId of target person group.",
+      "required": true,
+      "type": "string"
+     },
+     {
+      "$ref": "#/parameters/SubscriptionKey"
+     }
+    ],
+    "responses": {
+     "200": {
+      "description": "A successful call returns the person group's training status.",
+      "schema": {
+       "$ref": "#/definitions/TrainingStatus"
+      },
+      "examples": {
+       "application/json": {
+        "status": "succeeded",
+        "createdDateTime": "1/3/2017 4:11:35 AM",
+        "lastActionDateTime": null,
+        "message": null
+       }
+      }
+     },
+     "default": {
+      "description": "Error response.",
+      "schema": {
+       "$ref": "#/definitions/APIError"
+      }
+     }
+    },
+    "produces": [
+     "application/json"
+    ]
+   }
+  },
+  "/persongroups": {
+   "get": {
+    "description": "List person groups and their information.",
+    "operationId": "PersonGroups_List",
+    "parameters": [
+     {
+      "name": "start",
+      "in": "query",
+      "description": "List person groups from the least personGroupId greater than the \"start\". It contains no more than 64 characters. Default is empty. ",
+      "type": "string",
+      "enum": [
+       ""
+      ]
+     },
+     {
+      "name": "top",
+      "in": "query",
+      "description": "The number of person groups to list, ranging in [1, 1000]. Default is 1000. ",
+      "type": "integer",
+      "default": "1000",
```

Comment:
This should not have an `enum` -- that's used for single values. you can put a range in place with 
``` json
    "minimum": 1,
    "maximum": 1000,
```